test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1256,7 +1256,7 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
               expect(spyCalls.mostRecent().args).toEqual([BEACON_LOADING_CLS, false]);
             });
         });
-        itp('does not show beacon-loading animation when primaryAuth fails', function () {
+        it('does not show beacon-loading animation when primaryAuth fails', function () {
```

Comment:
why are we updating this ?

Reply:
I don't remember. I'll change it back and see if it affects anything.